lib/mayaUsd/ufe/UsdSceneItem.cpp

```diff
@@ -53,7 +53,13 @@ std::string UsdSceneItem::nodeType() const
 std::vector<std::string> UsdSceneItem::ancestorNodeTypes() const
 {
 	std::vector<std::string> strAncestorTypes;
-
+	
+	// during drag/drop in outliner UsdPrim can become invalidated which
+	// should never happens.
+	// MAYA-106644: HS August 31,2020 investigate the invalidation.
+	if (!fPrim.IsValid()) {
+		return {};
+	}
```

Comment:
@seando-adsk I added this temporarily workaround to pass the unexpected crash with my recent changes.

Reply:
This was a bug that I recently fixed with commit 4fbdf7f2a4f12fae6bcaef634faff3b3c626d198. 

I would argue that we should also not have checks like this. If the prim is invalid, then this entire UsdSceneItem is invalid and something has gone wrong. It should crash (or at least assert) so we can find and fix the real problem.

Reply:
@seando-adsk  Yes, this was just a temporarily check so I can investigate the issue with the new recent changes both in Maya and maya-usd. This block of code will disappear soon today.  Just ignore it.

I am looking into this as we speak and the problem is that in the middle of composite notification, your newly added **iconAt()** is getting called which shouldn't happen. We can talk internally to show you how this happens.

Reply:
addressed in 3fa7e6a9d95276e1fb7ca02d28cac92ca9995f59